src/components/EarnDashboard/components/StakingPositions/LpPositions.tsx

```diff
@@ -70,14 +70,18 @@ export const LpPositions: React.FC<ProviderPositionProps> = ({ ids, assetId }) =
         return
       }
 
-      mixpanel?.track(MixPanelEvents.ClickOpportunity, {
-        provider,
-        type,
-        assets: opportunity.original.underlyingAssetIds.map(assetId =>
-          getMaybeCompositeAssetSymbol(assetId),
-        ),
-        element: 'Table Row',
-      })
+      mixpanel?.track(
```

Comment:
can we just swap all of these out for the helpers now? super messy/verbose in the view layer

Reply:
yes